### PR TITLE
Get details

### LINKED ISSUE #22: orgs.getDetails function
Issue has been covered here:
https://github.com/emfoundation/ce100-app/issues/395

### PR DESCRIPTION
**Pull request by jay-meister**
closes #22

### CODE REVIEW

**Review by SimonLab — APPROVED** [review]
Looks good @JMurphyWeb , nice query :wink:

File: lib/formatters/orgsGetDetails.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  var challenges = {}; // { challenge_id: {id, title, tags: [ID, ID, ID]}, challenge_id: {id, title, tags: [ID, ID, ID]}}
+  var challenges_order = [];
+  rows.forEach(function (row) {
+    // if this is the first row, create the org and primary objects
+    if (!org.id) {
+      org.id = row.org_id;
+      org.name = row.org_name;
+      org.logo_url = row.org_logo_url;
+      org.mission_statement = row.org_mission_statement;
+      primary.first_name = row.primary_first_name;
```
Comment (mk4111) [inline lib/formatters/orgsGetDetails.js:181]: if an organisation had no primary user attached to this, would `row.primary_first_name` (and the other primary related keys in the row object) just return `null`?

File: lib/queries/orgs/getDetails.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    'JOIN',
+    'tags_organisations',
+    'ON',
+    'organisations.id = tags_organisations.organisations_id',
+    'JOIN',
+    'tags',
+    'ON',
+    'tags_organisations.tags_id = tags.id',
+    ';'
+  // -- WHEN WE JOIN THE INNER AND THE OUTER, WE GET 'REPEATED' RESULTS
```
Comment (mk4111) [inline lib/queries/orgs/getDetails.js:65]: What do you mean by repeated results? Sorry I'm not getting this... :S

Also, could you comment just above the commented out query below, what the difference is between that and the query above?

**Review by mk4111 — COMMENTED** [review]
This is suuuch a beast! @JMurphyWeb you baws!!

**Pull request merged**